client/models.py

```diff
@@ -63,25 +68,41 @@ class Volunteer(Timestampable):
         ("RAD_TV", "רדיו וטלוויזיה"),
         ("OTHR", "אחר")
     )
+    TYPES = (
+        ("NIGHBORHOOD_COORDINATOR", "רכז שכונה"),
+        ("CITY_COORDINATOR", "רכז עיר"),
+        ("STAFF", "מטה"),
+        ("HAMAL", "חמל"),
+        ("PROJECT", "פרויקט"),
+        ("CHILD_CARE", "משפחתון"),
+        ("MISSIONS", "משימות")
+    )
 
-    tz_number = models.CharField(max_length=11, blank=True)
-    full_name = models.CharField(max_length=200)
+    tz_number = models.CharField(max_length=ID_LENGTH, blank=True)
+    first_name = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, default="abcdef")
+    last_name = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, default="abcdef")
+    full_name = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, blank=True)
+    organization = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, blank=True)
     age = models.IntegerField()
+    volunteer_type = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, choices=TYPES, default="MISSIONS")
     areas = models.ManyToManyField(Area)
     languages = models.ManyToManyField(Language)
-    phone_number = models.CharField(max_length=200)
-    email = models.CharField(max_length=200, blank=True)
+    phone_number = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH)
```

Comment:
shouldn't phone field be limited to phone number size? (10)
```suggestion
    phone_number = models.CharField(max_length=10)
```

Reply:
you can use constant if you prefer

Reply:
I'm not sure why the original max length was 200 - figured maybe they wanted to allow international numbers, organization numbers, or some other unusual cases. Either way I didn't want to change the model without entirely understanding it, I just wanted to get some of those magic numbers out of the code

Reply:
Gotcha,
can you open issue for that? we might want to handle this on the future